previews/azure-functions-service-bus.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Azure Functions with Azure Service Bus
+component: ASBFunctions
+summary: Azure Functions Preview
+related:
+ - samples/previews/azure-functions
+reviewed: 2020-07-06
+---
+
+Host NServiceBus endpoints with [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/).
+
+## Basic Usage
+
+### Endpoint configuration
+
+snippet: endpoint-configuration
+
+The endpoint is automatically configured with the endpoint name and the transport connection string based on the values defined in the `[ServiceBusTrigger]` attribute using the `ServiceBusTriggeredEndpointConfiguration.FromAttributes` method.
+
+Alternatively, the endpoint name can be passed in manually:
+
+snippet: alternative-endpoint-setup
+
+### Azure Function definition
+
+snippet: function-definition
+
+## Configuration
+
+## License
+
+The license can be provided via the `NSERVICEBUS_LICENSE` environment variable which can be set via the Function settings in the Azure Portal.
+For local development, `local.settings.json` can be used. In Azure, specify a Function setting using environment variable as the key.
```

Comment:
```suggestion
For local development, `local.settings.json` can be used.
```

that part feels redundant given the first sentence?

Also, given local development is free now, is it still worth to document the local.settings.json file approach?

Reply:
> that part feels redundant given the first sentence?

Don't quite agree. In Azure there's still an option of `settings.json` file. The intent was to point out that a file approach is not necessary.

> Also, given local development is free now, is it still worth to document the local.settings.json file approach?

I'm not sure what do you mean by this. The cost of local development was never a consideration. Could you elaborate?